proto/expr.proto

```diff
@@ -224,9 +228,11 @@ message ExprNode {
     // jsonb ->> int, jsonb ->> text that returns text
     JSONB_ACCESS_STR = 601;
     // jsonb #> text[] -> jsonb
-    JSONB_EXTRACT_PATH = 613;
+    JSONB_EXTRACT_PATH = 626;
+    JSONB_EXTRACT_PATH_VARIADIC = 613;
```

Comment:
Could you explain the safety of this change?

Reply:
So old impl is already variadic?
<img width="551" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/4c55f903-e0a6-470d-9c90-6d37bfd2644c">